config/pipeline.yaml

```diff
@@ -3345,7 +3333,7 @@ build_configs:
 
   peterz:
     tree: peterz
-    branch: 'kernelci'
+    branch: 'perf/core'
```

Comment:
Sent email asking the Peter about it. Let's wait before merging the PR.

Reply:
Sent reminder

Reply:
I think remove tree and branches for now, if we get feedback we will add tree back

Reply:
Sure.